### PR TITLE
Configuration Update - Shown Email

### PR DESCRIPTION
**Pull request by KittenPixel**
This version will allow the private server owner to show a different email then "@grasscutter.io" if they want.

## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.

## No fixes.

- [ ] Bug fix
- [x] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/game/Account.java [hunk 1]
```diff
@@ -235,7 +235,7 @@ public void onLoad(Document document) {
             this.addPermission("*");
         }
 
-        // Set account default language as server default language
+        // Set account default language as fserver default language
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/game/Account.java]:
```suggestion
        // Set account default language to server default language
```

File: src/main/java/emu/grasscutter/game/Account.java [hunk 2]
```diff
@@ -109,7 +109,7 @@ public String getEmail() {
             return email;
         } else {
             // As of game version 3.5+, only the email is displayed to a user.
-            return this.getUsername() + "@grasscutter.io";
+            return this.getUsername() + "@"+ACCOUNT.playeremail;
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/game/Account.java]:
```suggestion
            return this.getUsername() + "@" + ACCOUNT.playerEmail;
```

File: src/main/java/emu/grasscutter/config/ConfigContainer.java [hunk 3]
```diff
@@ -140,6 +140,7 @@ public static class Account {
         public boolean autoCreate = false;
         public boolean EXPERIMENTAL_RealPassword = false;
         public String[] defaultPermissions = {};
+        public String playeremail = "grasscutter.io";
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/config/ConfigContainer.java]:
```suggestion
        public String playerEmail = "grasscutter.io";
```

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]

**Comment by KittenPixel** [comment]
Committed your requested changes King.

**Comment by KittenPixel** [comment]
So uhh anyone else gonna check it?

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**